iguana/m_notary_3rdparty

```diff
@@ -8,6 +8,8 @@ stdbuf -oL $1 ../agents/iguana 3rd_party &#> iguana.log 2> error.log  &
 
 myip=$(curl -s4 checkip.amazonaws.com)
 source pubkey.txt
+source pubkey_3p.txt
+echo "Using pubkey [$pubkey] for 3rd party iguana..."
 
```

Comment:
Why do we need these two lines? In earlier manuals, we didn't have any `pubkey_3p.txt`. Each node type, whether `main` or `3p`, has its own `pubkey.txt` file. Let's stick with this approach.